frontend/src/metabase/common/components/EntityPicker/components/EntityPickerModal/EntityPickerModal.tsx

```diff
@@ -264,3 +271,23 @@ const assertValidProps = (
     );
   }
 };
+
+const EntityPickerLoadingSkeleton = () => (
+  <Box data-testid="loading-indicator">
+    <Flex px="2rem" gap="1.5rem" mb="3.5rem">
+      <Repeat times={3}>
+        <Skeleton h="2rem" w="5rem" mb="0.5rem" />
```

Comment:
Where are the hex colors for the background colors for the Skeleton defined? Ideally they'd vascilate between `bg-light` and `bg-medium` rather than the sort of default gray they seem to be using.